$NetBSD: patch-aq,v 1.1 2010/06/12 19:36:01 dholland Exp $

Use NULL, not 0, to terminate execl() args. Otherwise (e.g. on LP64)
you might not get a pointer-sized zero value.

--- src/common.c.orig	2001-03-12 18:16:36.000000000 +0000
+++ src/common.c
@@ -859,7 +859,7 @@ void runcommand(cmd)
 		(void) dup(fd[PIPE_WRITE]);
 		(void) close(fd[PIPE_READ]);
 		(void) close(fd[PIPE_WRITE]);
-		(void) execl(_PATH_BSHELL, "sh", "-c", cmd, 0);
+		(void) execl(_PATH_BSHELL, "sh", "-c", cmd, NULL);
 		_exit(127);
 	}
 	(void) close(fd[PIPE_WRITE]);
